src/VisualStudio/Setup/source.extension.vsixmanifest

```diff
@@ -4,13 +4,38 @@
     <Identity Id="0b5e8ddb-f12d-4131-a71d-77acc26a798f" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Language Services</DisplayName>
     <Description>C# and VB.NET language services for Visual Studio.</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\Language Services</ShortcutPath>
   </Metadata>
   <Installation Experimental="true">
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.Pro" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinDesktopExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
+  <Installer>
+    <Actions>
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Workspaces.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Workspaces.Desktop.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.Features.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.Features.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.EditorFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.Features.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.Workspaces.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.EditorFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.Text.dll" />
+      <Action Type="Ngen" Path="Microsoft.VisualStudio.LanguageServices.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.dll" />
```

Comment:
Currently they are duplicated - one in the msbuild MSI and one here.  Not sure if we'll be able to remove that duplication with the app-local msbuild, but it would be nice if so.